client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -103,47 +128,56 @@ const GSuiteNewUser: FunctionComponent< Props > = ( {
 					} }
 					value={ domain }
 				/>
-			</Fragment>
+			</LabelWrapper>
 		);
 	};
 
 	return (
 		<div className="gsuite-new-user-list__new-user">
 			<FormFieldset>
-				<div className="gsuite-new-user-list__new-user-section">
+				<div
+					className={ classnames(
+						'gsuite-new-user-list__new-user-section',
+						showLabels ? 'show-labels' : null
+					) }
```

Comment:
It's probably simpler to use `classNames` in this format.

```suggestion
					className={ classNames(
						'gsuite-new-user-list__new-user-section',
						{  'show-labels' :  showLabels }
					) }
```

Reply:
Thanks for the suggestion!
I adopted that format and realised this class should be added one level up.